lib/ProductOpener/Export.pm

```diff
@@ -280,6 +280,26 @@ sub export_csv ($args_ref) {
 						}
 					}
 				}
+				elsif ($group_id eq "packaging") {
+					# packaging data will be exported in the CSV file in columns named like packaging_1_number_of_units
+					if (defined $product_ref->{packagings}) {
+						my $i = 0;    # number of the packaging component
+						foreach my $packaging_ref (@{$product_ref->{packagings}}) {
+							$i++;
+							my $j = 0;    # number of the field
+							foreach my $field (
+								qw(number_of_units shape material recycling quantity_per_unit weight_specified weight_measured)
+								)
+							{
+								$j++;
+								if (defined $packaging_ref->{$field}) {
+									my $field_sort_key = sprintf("%08d", $group_number * 1000 + $i * 10 + $j);
```

Comment:
can you add a comment to explain this $field_sort_key ?
(also above).

also why don't you use $field_sort_key . "_$j" like on line 272 ?

Reply:
There's a comment a bit above:

```
		# %populated_fields will contain the field name as the key,
		# and a sort key as the value so that the CSV columns are in the order of $options{import_export_fields_groups}
		my %populated_fields = ();
```

Reply:
I'll add another comment for packagings:
```

									# Generate a sort key so that the packaging fields in the CSV file are in this order:
									# - all fields for packaging component 1, then all fields for packaging component 2
									# - number_of_units, shape, material etc. (same a for loop above)
```